src/darwin/CHIPTool/CHIPTool/View Controllers/Temperature Sensor/TemperatureSensorViewController.m

```diff
@@ -250,8 +250,9 @@ - (void)reportFromUserEnteredSettings
                     errorHandler:^(NSError * error) {
                         NSLog(@"Status: update reportAttributeMeasuredValue completed with error %@", [error description]);
                     }
-                    subscriptionEstablished:^ {
-                    }];
+                    subscriptionEstablished:^{
+                    }
```

Comment:
Not your changes but I assume we could have used `nil` instead of an empty block for `subscriptionEstablished`

Reply:
Probably, yes...  Might predate that being nullable?